scripts/curate_basel_additional_data_20221028.py

```diff
@@ -0,0 +1,151 @@
+import os
+import shutil
+import json
+import argparse
+
+
+### make sure not to duplicate same patients from basel_mp2rage 
+other = '../../basel-mp2rage'
```

Comment:
this means the script needs to run from a specific location, which should be indicated in the comments above